crowd-server/crowd_server/apps/food_fact/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import QuestionView
+
+urlpatterns = [
+    path('food_fact/',QuestionView.as_view())
```

Comment:
Change the food_fact/ path name because it is duplicated. (http://localhost:9000/food_fact/food_fact/)